app/assets/stylesheets/components/lesson/new_submission_button.scss

```diff
@@ -0,0 +1,5 @@
+@include media-breakpoint-up(lg) {
```

Comment:
I don't think this belongs in its own component, imo we should wrap the button in an element and let that take care containing the size of the button. Checkout the contributing page buttons to get an idea of how we do that elsewhere https://www.theodinproject.com/contributing